doc/source/conf.py

```diff
@@ -52,13 +52,18 @@
     'sphinx.ext.intersphinx'
 ]
 
+# Which version of other SpiNNaker docs do we refer to?
+spinnaker_doc_version = "latest"
```

Comment:
I dont like this.

If you have to set the branch/tag somewhere why not just do it directly.

Reply:
Why update multiple places when we can have a single source of truth?

Reply:
This is a trade off between readability and number of places to updates.  My thinking was more from the prospective of using a script to update and then one change or many is the same effort.

On the other hand if someone else wants to do a manual update then yes a single source is better.